templates/incl-adventure-tabs.html

```diff
@@ -123,19 +123,27 @@ <h2 class="z-10 my-0 ml-auto pt-8 text-red-500 text-base">{{ _('specific_adventu
         </script>
 
         <div data-tabtarget="{{parsons['tab_name']}}" class="hidden">
-            <div>
-                {{ parsons['text']|commonmark }}
-
-
-                {% for key, line in parsons.code_lines.items() %}
-                    <div id="div1">
+    
+            <div class="flex flex-row">
+                <div class="{% if example_code %}w-1/2{% else %}w-full{% endif %} p-2 ltr:mr-2 rtl:ml-2">
```

Comment:
We only need these two lines if we have some "example_code" and want to split the panel into two views horizontally. An explanation on the left and the code examples on the right (as we do with adventures). Otherwise this doesn't need to be a `flex flex-row` and we don't need the `if` statement as well.